src/interface/events.ts

```diff
@@ -2,11 +2,14 @@
  * Interface specific events.
  */
 export enum InterfaceEventName {
+  ACCOUNT_DROPDOWN_BUTTON_CLICKED = 'Account Dropdown Button Clicked',
   APPROVE_TOKEN_TXN_SUBMITTED = 'Approve Token Transaction Submitted',
   CONNECT_WALLET_BUTTON_CLICKED = 'Connect Wallet Button Clicked',
   EXPLORE_BANNER_CLICKED = 'Explore Banner Clicked',
   EXPLORE_SEARCH_SELECTED = 'Explore Search Selected',
   EXPLORE_TOKEN_ROW_CLICKED = 'Explore Token Row Clicked',
+  FIAT_ONRAMP_BANNER_CLICKED = 'Fiat Onramp Banner Clicked',
+  FIAT_ONRAMP_WIDGET_OPENED = 'Fiat OnRamp Widget Opened',
```

Comment:
Use the same casing for Onramp throughout. If the enum value is ONRAMP then use Onramp, else ON_RAMP.